Assets/Plugins/LeapMotion/Modules/Hands/Scripts/HandBinder/HandBinderAutoBinder.cs

```diff
@@ -77,31 +74,28 @@ public static class HandBinderAutoRigger {
         }
 
         /// <summary>
-        /// The Autorigger uses this to select the children that match the finger definitions
+        /// The Autobinder uses this to select the children that match the finger definitions
         /// </summary>
         /// <param name="children">The found Children</param>
         /// <param name="definitions">The definition to sort through the children</param>
         /// <param name="isThumb">is this a thumb?</param>
         /// <returns></returns>
-        private static Transform[] SelectBones(List<Transform> children, string[] definitions, bool isThumb = false) {
-            //Can only ever be 4 bones per hand
-            var bones = new Transform[4];
-            int foundBoneIndex = 0;
+        private static Transform[] SelectBones(List<Transform> children, string[] definitions) {
```

Comment:
Is there a reason why the definitions are defined by strings?

Reply:
I think the parameter name definition is a bit odd here. Might be something that's clearer?